File: java/iroha_android/src/main/java/org/hyperledger/iroha/android/crypto/keystore/attestation/AttestationVerificationException.java
package org.hyperledger.iroha.android.crypto.keystore.attestation;

/**
 * Raised when attestation material cannot be validated or does not match the caller's policy.
 */
public final class AttestationVerificationException extends Exception {

  private static final long serialVersionUID = 1L;

  public AttestationVerificationException(final String message) {
    super(message);
  }

  public AttestationVerificationException(final String message, final Throwable cause) {
    super(message, cause);
  }
}

File: kotlin/client-android/src/main/java/org/hyperledger/iroha/sdk/crypto/keystore/attestation/AttestationVerificationException.kt
package org.hyperledger.iroha.sdk.crypto.keystore.attestation

/**
 * Raised when attestation material cannot be validated or does not match the caller's policy.
 */
class AttestationVerificationException : Exception {

    constructor(message: String) : super(message)

    constructor(message: String, cause: Throwable) : super(message, cause)

    companion object {
        private const val serialVersionUID: Long = 1L
    }
}
